templates/publisher/publicise/embedded_cards.html

```diff
@@ -57,18 +57,11 @@ <h4>Promote your snap using embeddable responsive card</h4>
       <label>HTML:</label>
     </div>
     <div class="col-7">
-      <div class="p-code-copyable not-cli">
-        <code class="p-code-copyable__input" id="snippet-card-html">&lt;iframe src="https://snapcraft.io/{{ snap_name }}/embedded?button=black&channels=true&summary=true&screenshot=true" frameborder="0" width="100%" height="320px" style="border: 1px solid #CCC; border-radius: 2px;" &gt;&lt;/iframe&gt;</code>
-        <button class="p-code-copyable__action js-clipboard-copy" data-clipboard-target="#snippet-card-html">Copy to clipboard</button>
-      </div>
+      <pre style="white-space: pre-line;"><code id="snippet-card-html">&lt;iframe src="https://snapcraft.io/{{ snap_name }}/embedded?button=black&channels=true&summary=true&screenshot=true" frameborder="0" width="100%" height="320px" style="border: 1px solid #CCC; border-radius: 2px;"&gt;&lt;/iframe&gt;</code></pre>
```

Comment:
Again, `id` quite possibly not needed anymore.

Reply:
this one is needed to update the content of the code snippet when you change what to show in the card.